src/webcore/value.rs

```diff
@@ -1291,4 +1297,50 @@ mod tests {
         let a: Result< (), _ > = js! { return 1; }.try_into();
         assert!( a.is_err() );
     }
+
+    #[test]
+    fn reference_stable() {
+        js! { Module.__test = {}; }
+        let a = js! { return Module.__test; }.as_reference().unwrap().as_raw();
+        let b = js! { return Module.__test; }.as_reference().unwrap().as_raw();
+        assert_eq!(a, b);
+
+        js! { delete Module.__test; }
+        js! { Module.__test = {}; }
+        let c = js! { return Module.__test; }.as_reference().unwrap().as_raw();
+        assert_ne!(a, c);
+
+        js! { Module.__test = {}; }
+    }
+
+    fn assert_noref(refid: i32) {
```

Comment:
I think this would be better as a function which returns `bool` - then you could test the inverse. (Also, perhaps a more descriptive name? `is_known_reference`? These are the tests, so we don't care if it's long.)

Reply:
Fixed.